io.github.DamnedAngel.msx-tile-forge.yaml

```
@@ -0,0 +1,39 @@
+app-id: io.github.DamnedAngel.msx-tile-forge
+runtime: org.freedesktop.Platform
+runtime-version: "25.08"
+sdk: org.freedesktop.Sdk
+
+command: msxtileforge
+
+finish-args:
+  - --share=ipc # needed for X11
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --share=network
+  - --socket=pulseaudio
+  - --device=dri
+  - --filesystem=xdg-documents
+
+modules:
+  - name: msx-tile-forge
+    buildsystem: simple
+    sources:
+      - type: archive
+        url: https://github.com/DamnedAngel/msx-tile-forge/releases/download/v01.00.00_rc20_5/msxtileforge_01.00.00_rc20_5_lin.tar.gz
+        sha256: d7090825cc51064c0d7dec93a61f5ecc607f4399c67ce823692f95becbfa39d5
+      - type: git
+        url: https://github.com/amaurycarvalho/msx-tile-forge.git
+        tag: feature/flatpak
+        commit: c58e028d60b516a2a269f43e5fa6512a8eb6296b
+    build-commands:
+      # Install Python dependencies (tkinter, numpy, scipy, Pillow, platformdirs, tqdm)
+      - mkdir -p /app/bin/_internal
+      - cp -r _internal/* /app/bin/_internal
+      # Install application
+      - install -Dm755 msxtileforge /app/bin/msxtileforge
+      # Install icon
+      - install -Dm644 flatpak/io.github.DamnedAngel.msx-tile-forge.png /app/share/icons/hicolor/256x256/apps/io.github.DamnedAngel.msx-tile-forge.png
+      # Install desktop entry
+      - install -Dm644 flatpak/io.github.DamnedAngel.msx-tile-forge.desktop /app/share/applications/io.github.DamnedAngel.msx-tile-forge.desktop
+      # Install AppStream metadata
```

```suggestion
      - mkdir -p /app/bin/_internal
      - cp -r _internal/* /app/bin/_internal
      - install -Dm755 msxtileforge /app/bin/msxtileforge
      - install -Dm644 flatpak/io.github.DamnedAngel.msx-tile-forge.png /app/share/icons/hicolor/256x256/apps/io.github.DamnedAngel.msx-tile-forge.png
      - install -Dm644 flatpak/io.github.DamnedAngel.msx-tile-forge.desktop /app/share/applications/io.github.DamnedAngel.msx-tile-forge.desktop
```

when I see these comment I read "water is wet"